src/BPEssentials/ExtensionMethods/String.cs

```diff
@@ -19,5 +19,10 @@ public static string Stringify(this bool val, string ifTrue, string ifFalse)
         {
             return val ? ifTrue : ifFalse;
         }
+
+        public static string CleanerMessage(this string san)
```

Comment:
Suggested to be renamed to something more user friendly, like `CleanMessage()`, `SanitizeMessage()`, `SanitizeString()`, `CleanString()`, etc. I myself like `SanitizeString`, as it clearly explains what it will do.
```suggestion
        public static string SanitizeString(this string san)
```

Reply:
CleanMessage() is currently the BP native Sanitizer, but it's just replacing & and < and > to ^, so CleanerMessage() is the nicer version of it.

Reply:
Alright. I'll merge the PR.